compiler/backend/src/org/jetbrains/kotlin/codegen/state/KotlinTypeMapper.kt

```diff
@@ -1199,6 +1199,29 @@ class KotlinTypeMapper @JvmOverloads constructor(
             ?: computeInternalName(classDescriptor, typeMappingConfiguration, isIrBackend)
     }
 
+    private fun getDefaultDescriptor(
```

Comment:
Could you keep old position of method and just pass neccesery parameters to it: actually there would be new TypeMapper for IR so all IR related logic would be removed from current one

Reply:
Will do.